metadata-ingestion/setup.py

```diff
@@ -805,6 +805,7 @@
         "datahub-gc = datahub.ingestion.source.gc.datahub_gc:DataHubGcSource",
         "datahub-debug = datahub.ingestion.source.debug.datahub_debug:DataHubDebugSource",
         "datahub-apply = datahub.ingestion.source.apply.datahub_apply:DataHubApplySource",
+        "datahub-mock-data = datahub.ingestion.source.mock_data.datahub_mock_data:DataHubMockDataSource",
```

Comment:
Having mechanisms to generate mock data is definitely valuable.
What concerns me is coupling that logic directly with an ingestion source.

Would it make more sense to have a general-purpose DataHub source that ingests arbitrary MCE file(s)? That feels more flexible and reusable.

Reply:
-  Ingesting arbitrary MCE can already be done using `datahub ingest mcps FILE_NAME`
- There is already a datahub source which is for migration.

Using the ingestion framework for this allows us to test for various bugs in here and iterate on ingestion framework items which isn't really possible otherwise. This is a separate isolated source which is clearly marked as being there for testing. Unless there are very strong opinions would request this be allowed in.